docs/40-troubleshooting.md

```diff
@@ -0,0 +1,19 @@
+# Getting logs without Azure Portal
+## Viewing files from Azure Storage
+In every foreach run, the job will push its logs into Azure Storage that can be fetched by the user. For more information on reading log files, check out [managing storage](./41-managing-storage-via-R.md). 
+
+## Viewing files directly from compute node
+Logs on a cluster node are not persisted through Azure Storage. `getClusterFile` function will fetch any files including stdout and stderr log files in the cluster. This is particularly useful for users that utilizing [customize script](./30-customize-cluster.md) on their nodes and installing specific [packages](./20-package-management.md).
+
+```R
+# This will download stderr.txt directly from the cluster. 
+getClusterFile(cluster, "tvm-1170471534_2-20170829t072146z", "stderr.txt", downloadPath = "pool-errors.txt")
```

Comment:
Add docs before this for every file type the user can download. For example:
Cluster log setup files include

File name | Description
--- | ---
stdout.txt | Contains the standard output of files. This includes any additional logging done during cluster setup time
stderr.txt | Contains the verbose and error logging during cluster setup